app/code/Magento/InventoryBundle/Plugin/InventoryConfiguration/IsSourceItemManagementAllowedForProductType/DisableBundleTypePlugin.php

```diff
@@ -5,25 +5,25 @@
  */
 declare(strict_types=1);
 
-namespace Magento\InventoryBundle\Plugin\InventoryConfiguration\IsSourceItemsAllowedForProductType;
+namespace Magento\InventoryBundle\Plugin\InventoryConfiguration\IsSourceItemManagementAllowedForProductType;
```

Comment:
it's better to fix namespace as well 
`Magento\InventoryBundle\Plugin\InventoryConfiguration\..` -> `Magento\InventoryBundle\Plugin\InventoryConfigurationApi\`